packages/suite/src/views/onboarding/steps/SelectBackupType.tsx

```diff
@@ -189,7 +202,7 @@ type OptionProps = {
 };
 
 const Option = ({ children, onSelect, isChecked, 'data-test': dataTest }: OptionProps) => (
-    <OptionStyled onClick={onSelect}>
+    <OptionStyled onClick={onSelect} $hasHoverInteraction={true}>
```

Comment:
Why not to have hover interaction by defaut turned on?

Reply:
This is used for "selected" element as well. We dont want it there.